libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Cert.hs

```diff
@@ -25,8 +25,11 @@ certEnvSpec ::
   Specification fn (CertEnv (ConwayEra StandardCrypto))
 certEnvSpec =
   constrained $ \ce ->
-    match ce $ \_ pp _ _ _ ->
-      satisfies pp pparamsSpec
+    match ce $ \_ pp _ a b ->
+      [ satisfies pp pparamsSpec
+      , assert $ a ==. lit SNothing
+      , assert $ sizeOf_ b ==. 0
```

Comment:
I'm not sure why these constraints are necessary

Reply:
They are not necessary. In fact they are never looked at, so any value will do.
But again when there is a failure, they hide what is really wrong in monstrous sea of irrelevant detail. So I am going with these

 [ satisfies pp pparamsSpec
 , assert $ genHint 3 proposals
 ]